react/react1/week2/react1-week2/src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import Main from "./Main";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <React.StrictMode>
+    <Main />
```

Comment:
if you're not using the conventional App.js, please remove it and its imports, so we don't get confused